lnwire/custom_records.go

```diff
@@ -14,6 +14,26 @@ const (
 	MinCustomRecordsTlvType = 65536
 )
 
+// ParseCustomRecords decodes a set of custom records from a byte slice.
+func ParseCustomRecords(b tlv.Blob) (CustomRecords, error) {
```

Comment:
Looks like this ended up a duplicate of `NewCustomRecordsFromTlvBlob` below. So can just use that.
Or maybe rename `NewCustomRecordsFromTlvBlob` to `ParseCustomRecords` which is way shorter.

Reply:
removed this, kept the latter and renamed it to match the original one